vendor/bat-native-ads/include/bat/ads/ad_content_info.h

```diff
@@ -35,6 +35,7 @@ struct ADS_EXPORT AdContentInfo final {
   std::string ToJson() const;
   bool FromJson(const std::string& json);
 
+ public:
```

Comment:
Why is this needed?

Reply:
Good catch as copy/paste which should not be in this PR

Reply:
Fixed